documentation-website/Writerside/topics/Transactions.md

```diff
@@ -0,0 +1,317 @@
+# Transactions
+
+## Overview
+
+CRUD operations in Exposed must be called from within a _transaction._ Transactions encapsulate a set of DSL operations.
+To create and execute a transaction with default parameters, simply pass a function block to the `transaction` function:
+
+```kotlin
+transaction {
+    // DSL/DAO operations go here
+}
+```
+
+Transactions are executed synchronously on the current thread, so they _will block_ other parts of your application! If
+you need to execute a transaction asynchronously, consider running it on a separate `Thread`.
+
+### Accessing returned values
+
+Although you can modify variables from your code within the transaction block, `transaction` supports returning a value
+directly, enabling immutability:
+
+```kotlin
+val jamesList = transaction {
+    Users.selectAll().where { Users.firstName eq "James" }.toList()
+}
+// jamesList is now a List<ResultRow> containing Users data
+```
+
+*Note:* `Blob` and `text` fields won't be available outside a transaction if you don't load them directly. For `text`
+fields you can also use the `eagerLoading` param when defining the Table to make the text fields available outside the
+transaction.
+
+```kotlin
+// without eagerLoading
+val idsAndContent = transaction {
+   Documents.selectAll().limit(10).map { it[Documents.id] to it[Documents.content] }
+}
+
+// with eagerLoading for text fields
+object Documents : Table() {
+  ...
+  val content = text("content", eagerLoading = true)
+}
+
+val documentsWithContent = transaction {
+   Documents.selectAll().limit(10)
+}
+```
+
+### Working with multiple databases
+
+_This functionality supported since 0.10.1 version_
```

Comment:
There's a few lines in this file (and others) that reference a very old version of Exposed.

Is there still value to the users in mentioning that things were different before a version as old as 0.10.1?
Here are some other mentions of versions older than 0.30.1:
- [Base Modules](https://jetbrains.github.io/Exposed/exposed-modules.html#base-modules) in Exposed Modules page is split into 2 subsections: one for before 0.18.1 and another for afterwards.
- [Conditional Where](https://jetbrains.github.io/Exposed/deep-dive-into-dsl.html#conditional-where) in Deep Dive into DSL has 2 code block examples for version 0.8.1 and older.
- [Eager Loading section](https://jetbrains.github.io/Exposed/deep-dive-into-dao.html#eager-loading) in Deep Dive into DAO mentions that it is available since version 0.13.1.

Reply:
Let's keep it until 1.0